gatorgrouper/utils/read_student_file.py

```diff
@@ -26,8 +26,11 @@ def read_csv_data(filepath):
             for value in record[1:]:
                 if value == "True":
                     temp.append(True)
-                elif value == "False":
-                    temp.append(False)
+                else:
+                    try:
+                       temp.append(float(value))
+                    except ValueError: # value cannot be converted to float
+                        pass
```

Comment:
Does this still handle `False` then?

Reply:
I don't think it does, this PR need to be reverted to the older version.

Reply:
Why reverted? It can probably be added here...